oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxSensors.java

```diff
@@ -62,8 +62,13 @@ public LinuxSensors() {
                 // Find any *_input files in that path
                 @Override
                 public boolean accept(File f) {
-                    return f.getName().startsWith(sensorPrefix) && f.getName().endsWith("_input")
-                            && FileUtil.getIntFromFile(f.getName()) > 0;
+                    try {
+                        return f.getName().startsWith(sensorPrefix) && f.getName().endsWith("_input")
+                                && FileUtil.getIntFromFile(f.getCanonicalPath()) > 0;
+                    }
+                    catch(Exception e) {
```

Comment:
I don't like catching all exceptions here. Can we narrow this to the actual exception thrown?  And log a message other than just returning?